lang/LangPrimSource/SC_ComPort.h

```diff
@@ -82,8 +82,8 @@ class UDP : private boost::noncopyable {
 
     int mPortNum;
     HandleDataFunc mHandleFunc;
-    static constexpr int receiveBufferSize = 8 * 1024 * 1024;
-    static constexpr int sendBufferSize = 8 * 1024 * 1024;
+    static constexpr int receiveBufferSize = 4 * 1024 * 1024;
+    static constexpr int sendBufferSize = 4 * 1024 * 1024;
```

Comment:
I think we should add `static constexpr int fallbackBufferSize = 1 * 1024 * 1024`, and perhaps move all these constexpr to `common/SC_OscUtils.hpp`?

Reply:
Actually, we could make a function `setUDPBufferSizes` in there, and call it from sclang/scsynth/supernova. But we could also do this in a subsequent refactoring PR

Reply:
> I think we should add static constexpr int fallbackBufferSize = 1 * 1024 * 1024

Most definitely.